classes/ActionScheduler_Compatibility.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Class ActionScheduler_Compatibility
+ */
+class ActionScheduler_Compatibility {
+
+	/**
+	 * Converts a shorthand byte value to an integer byte value.
+	 *
+	 * Wrapper for wp_convert_hr_to_bytes(), moved to load.php in WordPress 4.6 from media.php
+	 *
+	 * @link https://secure.php.net/manual/en/function.ini-get.php
+	 * @link https://secure.php.net/manual/en/faq.using.php#faq.using.shorthandbytes
+	 *
+	 * @param string $value A (PHP ini) byte value, either shorthand or ordinary.
+	 * @return int An integer byte value.
+	 */
+	public static function convert_hr_to_bytes( $value ) {
+		if ( function_exists( 'wp_convert_hr_to_bytes' ) ) {
+			return wp_convert_hr_to_bytes( $value );
+		} else {
+			$value = strtolower( trim( $value ) );
+			$bytes = (int) $value;
+
+			if ( false !== strpos( $value, 'g' ) ) {
+				$bytes *= GB_IN_BYTES;
+			} elseif ( false !== strpos( $value, 'm' ) ) {
+				$bytes *= MB_IN_BYTES;
+			} elseif ( false !== strpos( $value, 'k' ) ) {
+				$bytes *= KB_IN_BYTES;
+			}
+
+			// Deal with large (float) values which run into the maximum integer size.
+			return min( $bytes, PHP_INT_MAX );
+		}
+	}
+
+	/**
+	 * Attempts to raise the PHP memory limit for memory intensive processes.
+	 *
+	 * Only allows raising the existing limit and prevents lowering it.
+	 *
+	 * Wrapper for wp_raise_memory_limit(), added in WordPress v4.6.0
+	 *
+	 * @return bool|int|string The limit that was set or false on failure.
+	 */
+	public static function raise_memory_limit() {
+		if ( function_exists( 'wp_raise_memory_limit' ) ) {
+			return wp_raise_memory_limit( 'admin' );
+		} else {
```

Comment:
Putting all of this logic in an `else` block isn't necessary, since the function will return early if the WP function exists. It should be removed from the `else` block to help with readability by reducing indentation.